config/sync/rdf_entity.rdfentity.rdf_graph.yml

```diff
@@ -5,7 +5,7 @@ dependencies: {  }
 name: 'RDF Graph'
 rid: rdf_graph
 description: ''
-label_singular: RDF graph
-label_plural: RDF graphs
+label_singular: 'RDF graph'
+label_plural: 'RDF graphs'
 label_count:
-  default: "1 RDF graph\x03@count RDF graphs"
+  default: !!binary MSBSREYgZ3JhcGgDQGNvdW50IFJERiBncmFwaHM=
```

Comment:
I checked this and this value remains the same but is now represented in binary base64 encoded format because of a non-standard character present in the string.